dashboard/README.md

```diff
@@ -0,0 +1,21 @@
+# Bokeh dashboard for accessibility audit
+
+A dashboard with commonly used Bokeh plot elements for an initial accessibility audit.
```

Comment:
```suggestion

This repository contains assets and artefacts related to Bokeh accessibility audits:

- [A dashboard with commonly used Bokeh plot elements](./dashboard/dashboard.ipynb). This dashboard can be accessed at <https://quansight-labs.github.io/bokeh-a11y-audit/>.
```

Since we will end up holding more items than only the dashboard